test/samples/custom/items/item_body_dir_attr.xml

```diff
@@ -0,0 +1,26 @@
+<assessmentItem xmlns="http://www.imsglobal.org/xsd/imsqti_v2p2" xmlns:m="http://www.w3.org/1998/Math/MathML"
```

Comment:
Simply adding a file here doesn't create a test case, does it?

Reply:
Yes, sorry mixed with other repo which check in same time